### PR TITLE
add checkboxes to variables when in export

### PR DESCRIPTION
**Pull request by beaugogh**
TMT-568

### CODE REVIEW

File: src/app/modules/gb-side-panel-module/accordion-components/gb-variables/gb-variables.component.ts [hunk 1]
```diff
@@ -32,7 +40,23 @@ export class GbVariablesComponent implements OnInit {
         this.categorizedVariables.get(variable.type).push(variable);
       }
     });
-    console.log('categorized variables: ', this.categorizedVariables);
+  }
+
+  onCheckAll(e) {
+    this.checkAllText = e ? 'All selected' : 'None selected';
```
Comment (ewelinagr) [inline src/app/modules/gb-side-panel-module/accordion-components/gb-variables/gb-variables.component.ts]: Having "All selected" or "None selected" test above the variables is a bit confusing, when you select/deselect something not using `CheckAll` checkbox, but a checkbox next to a specific variable. Maybe better to have a counter there as well as a third option: "`x` selected" or at least hide the text at all, when not all/none is selected.

Comment (beaugogh) [reply src/app/modules/gb-side-panel-module/accordion-components/gb-variables/gb-variables.component.ts]: ah, right, never thought of it. will change it!

File: src/app/modules/gb-side-panel-module/accordion-components/gb-variables/gb-variables.component.ts [hunk 2]
```diff
@@ -40,19 +39,28 @@ export class GbVariablesComponent implements OnInit {
...
+  updateCheckAllText() {
+    let numSelected = 0;
+    this.categorizedVariables.forEach((list: Array<Concept>) => {
+      list.forEach((c: Concept) => {
+        if (c.selected) {
+          numSelected++;
+        }
+      });
+    });
+    this.checkAllText = `${numSelected} variables selected`;
```
Comment (ewelinagr) [inline src/app/modules/gb-side-panel-module/accordion-components/gb-variables/gb-variables.component.ts]: maybe variable(s)? to fit also numSelected = 1 :)

Comment (beaugogh) [reply src/app/modules/gb-side-panel-module/accordion-components/gb-variables/gb-variables.component.ts]: good point!

**Pull request merged**